packages/main/src/plugin/install/extension-installer.ts

```diff
@@ -120,16 +126,28 @@ export class ExtensionInstaller {
       return;
     }
 
-    const titleLabel = imageConfigLabels['org.opencontainers.image.title'];
+    const titleLabel = imageConfigLabels['org.opencontainers.image.title'] as string;
```

Comment:
```suggestion
    const titleLabel = imageConfigLabels['org.opencontainers.image.title'] as string | undefined;
```

Reply:
I guess you meant a `||` ?

a couple of lines after there is test on the availability so empty string or undefined string would act the same way

Reply:
> I guess you meant a `||` ?

No I mean this should not be cast as a string when it could be undefined ? the test after ensure it not undefined, therefore this can be either a string or undefined ?

After the check you mention it is a string, as it cannot be undefined, but before this it can be